ui/src/notebooks/shared/event.ts

```diff
@@ -0,0 +1,30 @@
+import {reportEvent, toNano} from 'src/cloud/utils/reporting'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+
+interface KeyValue {
+  [key: string]: string
+}
+
+export const event = (title: string, meta: KeyValue = {}): void => {
+  const time = toNano(Date.now())
+
+  if (isFlagEnabled('streamEvents')) {
+    /* eslint-disable no-console */
+    console.log(`Event:  [ ${title} ]`)
+    if (Object.keys(meta).length) {
+      console.log(
+        Object.entries(meta)
+          .map(([k, v]) => `        ${k}: ${v}`)
+          .join('\n')
+      )
+    }
+    /* eslint-enable no-console */
+  }
+
+  reportEvent({
+    timestamp: time,
+    measurement: title,
+    fields: {},
+    tags: {...meta},
+  })
```

Comment:
// src/shared/utils/analytics.ts
fireEvent(title, meta) would also send this to Google Analytics.

Reply:
but.. in src/cloud/utils right?